client/components/payment-activity/index.tsx

```diff
@@ -83,11 +97,53 @@ const PaymentActivity: React.FC = () => {
 		return <></>;
 	}
 
+	const options = Object.keys( dateRangePresets ).map( ( presetName ) => {
+		const preset = dateRangePresets[ presetName ];
+		return {
+			key: presetName,
+			name: preset.displayKey,
+			hint: formatDateRange( preset.start, preset.end ),
+		};
+	} );
+
 	return (
 		<Card>
-			<CardHeader>
+			<CardHeader className="wcpay-payment-activity__card__header">
 				{ __( 'Your payment activity', 'woocommerce-payments' ) }
-				{ /* Filters go here */ }
+				<InlineLabelSelect
+					label="Select an option"
+					options={ options }
+					value={ options.find(
+						( option ) =>
+							option.key === selectedDateRange.preset_name
+					) }
+					placeholder="Select an option..."
```

Comment:
💭 I suggest we use a more descriptive label, e.g. `Period`, in the case we're don't address #8928

```suggestion
					label="Period"
					options={ options }
					value={ options.find(
						( option ) =>
							option.key === selectedDateRange.preset_name
					) }
```

Reply:
Done here - [ca617a1](https://github.com/Automattic/woocommerce-payments/pull/8927/commits/ca617a155cd9c8786d949474c6083bc69e016ec9)